### PR TITLE
Fix plugin python3 compatibility

### PR DESCRIPTION
**Pull request by smclay**
When I build the MWA plugins for python 3 I get lots of compatibility errors. I had to fix the bugs before I can build the module. I am unsure why the travis pipeline does not see these errors. Anyway, the changes are quite minor.

### CODE REVIEW

**Comment by rtobar** [comment]
@smclay thanks for these contributions, it all looks very sensible.

The Travis pipeline doesn't check that the plug-ins under the different subdirectories under `ngamsPlugIns` can be imported, it only checks the files immediately under `ngamsPlugIns` can: https://github.com/ICRAR/ngas/blob/2ce3debacf0c15a7124962f5bc4c2d301148a029/.travis/run_tests.sh#L48-L57

The main idea is that plug-ins under the different subdirectories are not really *that* looked after, as they could/should be provided by each specific project separately from the NGAS source code repository. What we currently have there is for historical and practical purposes, but in the long run it would be better if project-specific plugins are maintained separately.

In other words: please don't feel any pressure to get these issues fixed if you see more in the future. If this is somehow affecting your builds the simplest alternative for you would be to remove these project-specific files from your builds and leave only the ones you need.
